github/messages_test.go

```diff
@@ -539,6 +539,11 @@ func TestParseWebHook(t *testing.T) {
 			payload:     &WorkflowRunEvent{},
 			messageType: "workflow_run",
 		},
+
```

Comment:
Ditto, and no need for a blank line here.

Reply:
fixed